aspnet/publishing/linuxproduction.rst

```diff
@@ -0,0 +1,190 @@
+Publish to a Linux Production Environment
+=========================================
+
+By `Sourabh Shirhatti`_
+
+In this guide, we will cover setting up a production-ready ASP.NET environment on an Ubuntu 14.04 Server. Though we will cover the specifics of using Ubuntu 14.04, the content will be applicable to other flavors of Linux.
+
+We will take an existing ASP.NET 5 application and place it behind a reverse-proxy server. We will then setup the reverse-proxy server to forward requests to our Kestrel web server.
+
+Additionally we will write a startup script to ensure our web application runs on startup as a daemon as well as a configure a process management tool to help restart our web application in the event of a crash to guarantee high availibility.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+Prerequisites
+-------------
+
+1. Acess to an Ubuntu 14.04 Server with a standard user account with
+   sudo privilege.
+2. An exisiting ASP.NET 5 application.
+
+Before getting started you require LibUV. Follow the instructions on the :doc:`/getting-started/installing-on-linux` page.
+
+Copy over your app
+------------------
+
+Run ``dnu publish --runtime dnx-coreclr-linux-x64.1.0.0-rc1-update1`` from your dev environment to package your
+application into a self-contained directory that can be run on your server.
+
+Before we proceed, go ahead and copy over your ASP.NET 5 application to your server using whatever tool (SCP, FTP, etc) integrates into your workflow. Try and run the app and navigate to ``http://<serveraddress>:<port>`` in your browser to see if the application runs fine on Linux. I recommend you have a working app before proceeding.
+
+If you do not have an application, I recommend using the `Yeoman generators <https://github.com/omnisharp/generator-aspnet>`_ to quickly scaffold a skeleton app.
+
+
+Configure a reverse proxy server
+--------------------------------
+
+A reverse proxy is a common setup for serving dynamic web applications. The reverse proxy terminates the HTTP request and forwards it to the ASP.NET application.
+
+Why use a reverse-proxy server?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Kestrel is great for serving dynamic content from ASP.NET, however the web serving parts aren’t as feature rich full-featured servers like IIS, Apache or Nginx. A reverse proxy-server can allow you to offload work like serving static content, caching requests, compressing requests, and SSL termination from the HTTP server. The reverse proxy server may reside on a dedicated machine or may be deployed alongside an HTTP server.
+
+For the purposes of this guide, we are going to use a single instance of Nginx that runs on the same server alongside your HTTP server. However, based on your requirements you may chose a different setup.
```

Comment:
Typo: "chose" rather than "choose". I'd suggest reversing the order of the sentence (ie. "However, you may choose a different setup based on your requirements"